cmd/bucket-replication.go

```diff
@@ -843,6 +851,40 @@ func replicateObject(ctx context.Context, ri ReplicateObjectInfo, objectAPI Obje
 	}
 }
 
+func replicateObjectWithMultipart(ctx context.Context, c *miniogo.Core, bucket, object string, r io.Reader, objInfo ObjectInfo, opts miniogo.PutObjectOptions) (uploadID string, err error) {
+	var uploadedParts []miniogo.CompletePart
+	uploadID, err = c.NewMultipartUpload(context.Background(), bucket, object, opts)
+	if err != nil {
+		return
+	}
```

Comment:
```suggestion
	}
	defer func() {
		// Abort on error...
		if err != nil {
			c.AbortMultipartUpload(ctx, bucket, object, uploadID)
		}
	}()

```

Reply:
already doing this here https://github.com/minio/minio/pull/12535/files#diff-5f4783bb565baf3aa0ffa68ea4808cbc9202f59cc4023cc85ec17ce029e7d2e0R785